tests/kernel/pipe/pipe/src/test_pipe.c

```diff
@@ -874,3 +880,81 @@ void test_pipe_put_min_xfer(void)
 	zassert_equal(res, 0, "did not write min_xfer");
 	zassert_true(bytes_written >= 1, "did not write min_xfer");
 }
+
+/**
+ * @brief Defining and initializing pipes at compile time
+ * @details Check if the kernel provided a mechanism for defining and
+ * initializing pipes at compile time(K_PIPE_DEFINE()).
+ * - Using the pipe which defined and initialized at compile time to
+ * transfer data and verify it.
+ * @ingroup kernel_pipe_tests
+ * @see K_PIPE_DEFINE()
```

Comment:
Uh... where is the K_PIPE_DEFINE() for this pipe?  If it was already in the code, then surely this test case was already covered?

Reply:
yes,  already covered by other test case,  thanks for pointing out.